internal/experiment/generation/stormforger.go

```diff
@@ -164,56 +188,70 @@ func (s *StormForgerSource) Metrics() ([]redskyv1beta1.Metric, error) {
 	return result, nil
 }
 
+func (s *StormForgerSource) stormForgerTestCase() (org, name string) {
+	parts := strings.Split(s.Scenario.StormForger.TestCase, "/")
```

Comment:
we do `strings.Split` because test case can either be a /path/to/file.js or http://url/to/file.js ?

Reply:
The `testCaseFile` can be a path or URL the JavaScript source, the `testCase` (what we are splitting here) is just the StormForger test case name (i.e. `<org>/<name>`). I believe there is another test case name format that involves more slashes and a UUID, but I'm not 100% sure.